scripts/fast-reboot

```diff
@@ -398,6 +398,19 @@ function save_counters_folder() {
     fi
 }
 
+function check_warm_restart_in_progress() {
+    sonic-db-cli STATE_DB keys "WARM_RESTART_ENABLE_TABLE|*" | while read key ; do
+        if [[ "$(sonic-db-cli STATE_DB hget $key enable)" = "true" ]]; then
```

Comment:
Suggest to use x"$(...)" == x"true"  instead. this is to avoid error when the sonic-db-cli return empty result.

Reply:
Thanks for this comment. Fixed as prescribed.